src/style.css

```diff
@@ -1 +1,39 @@
-/* add styles here */
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
```

Comment:
Consider adding a fallback font-family for better cross-browser compatibility. For example, if you're using a custom font for the `.stars` class, you should specify a generic font family as a fallback in case the custom font fails to load.